include/circt/Dialect/HW/HWTypes.td

```diff
@@ -115,6 +115,7 @@ def InnerSymProperties : AttrDef<HWDialect, "InnerSymProperties"> {
   let hasCustomAssemblyFormat = 1;
   // The assembly format is as follows:
   // "`<` `@` $name `,` $fieldID `,` $sym_visibility `>`";
+  let genVerifyDecl = 1;
```

Comment:
I wasn't aware of this. Cool!

Reply:
Me too, honestly, somehow thought this wasn't a thing!